akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/JournalInteractions.scala

```diff
@@ -111,8 +111,9 @@ private[akka] trait JournalInteractions[C, E, S] {
   }
 
   protected def internalSaveSnapshot(state: Running.RunningState[S]): Unit = {
```

Comment:
We declare return Unit but in fact it could throw. Should this be documented?

Reply:
`IllegalStateException` here would be a bug in Akka, we should not attempt to save null snapshots